From a2f9265d49dc000d5becf65da0ec42a92eac19cb Mon Sep 17 00:00:00 2001
From: Martin Stefany <martinstefany@users.noreply.github.com>
Date: Sun, 7 Jun 2015 15:39:12 +0200
Subject: [PATCH] support '-' (dash) in config file

support options like 'no-tls-verify', 'tls-ca-path' in config file
---
 sendxmpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sendxmpp b/sendxmpp
index 0d7c4ff..8b2bebe 100755
--- a/sendxmpp
+++ b/sendxmpp
@@ -155,7 +155,7 @@ sub read_config_file ($) { # {{{
 
 		#s/\#.*$//; # ignore comments in lines
 
-		if (/^([a-z]+):\s*(.*)$/) {
+		if (/^([a-z-]+):\s*(.*)$/) {
 			$config{$1} = $2;
 		}
 		# Hugo van der Kooij <hvdkooij AT vanderkooij.org> has account with '#' as username
